[2025-01-10T16:51:06.412+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-10T16:51:06.425+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: stock_information.append_to_database scheduled__2025-01-10T16:50:00+00:00 [queued]>
[2025-01-10T16:51:06.432+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: stock_information.append_to_database scheduled__2025-01-10T16:50:00+00:00 [queued]>
[2025-01-10T16:51:06.435+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-10T16:51:06.447+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): append_to_database> on 2025-01-10 16:50:00+00:00
[2025-01-10T16:51:06.453+0000] {standard_task_runner.py:72} INFO - Started process 537 to run task
[2025-01-10T16:51:06.456+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'stock_information', 'append_to_database', 'scheduled__2025-01-10T16:50:00+00:00', '--job-id', '95', '--raw', '--subdir', 'DAGS_FOLDER/vnstock_stream.py', '--cfg-path', '/tmp/tmp53kp3dkp']
[2025-01-10T16:51:06.458+0000] {standard_task_runner.py:105} INFO - Job 95: Subtask append_to_database
[2025-01-10T16:51:06.496+0000] {task_command.py:467} INFO - Running <TaskInstance: stock_information.append_to_database scheduled__2025-01-10T16:50:00+00:00 [running]> on host 453ac4defedc
[2025-01-10T16:51:06.570+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='TuanDz' AIRFLOW_CTX_DAG_ID='stock_information' AIRFLOW_CTX_TASK_ID='append_to_database' AIRFLOW_CTX_EXECUTION_DATE='2025-01-10T16:50:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-10T16:50:00+00:00'
[2025-01-10T16:51:06.572+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-10T16:51:21.608+0000] {logging_mixin.py:190} INFO - Lỗi kết nối db: ('HYT00', '[HYT00] [Microsoft][ODBC Driver 18 for SQL Server]Login timeout expired (0) (SQLDriverConnect)')
[2025-01-10T16:51:21.610+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-10T16:51:21.618+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-10T16:51:21.619+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=stock_information, task_id=append_to_database, run_id=scheduled__2025-01-10T16:50:00+00:00, execution_date=20250110T165000, start_date=20250110T165106, end_date=20250110T165121
[2025-01-10T16:51:21.645+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-10T16:51:21.658+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-10T16:51:21.662+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
